include/Core/BulkDelete.php

```diff
@@ -467,6 +472,22 @@ private function get_delete_metas_admin_page() {
 		return $metas_page;
 	}
 
+	/**
+	 * Get Bulk Delete Terms admin page.
```

Comment:
The Docblock comment should say what the function does and not what the should do. For example, "Get**s** Bulk Delete Term's admin page" is correct.